dds/DCPS/RcEventHandler.h

```diff
@@ -30,13 +30,13 @@ class RcEventHandler
 
   }
 
-  ACE_Event_Handler::Reference_Count add_reference()
+  virtual ACE_Event_Handler::Reference_Count add_reference()
```

Comment:
It is a virtual member function with or without the keyword `virtual`.  Is there any reason to add it?  Unlike `override` in C++11, it doesn't help find errors.

Reply:
I was trying to figure out what was going on with macOS and I found that it doesn't have any effect after trying it out, but forgot to remove it.